content/authorization/guides/system-vendor/system-user/democlient/_index.nb.md

```
@@ -0,0 +1,78 @@
+weight: 10
+---
+
+## Demoklient
+
+For en demo av hvordan leverandørstyrt opprettelse kan se ut, se vår demoklient [SmartCloud](http://smartcloudaltinn.azurewebsites.net).
+
+Se kode med dokumentasjon [her](https://github.com/TheTechArch/altinn-systemuser).
+
+For opprettelse av systembrukere kan testbrukere/organisasjoner fra Tenor benyttes.
+
+## Testing av systembruker i TT02
```

markdownlint-cli2 (0.18.1)

Link text should be descriptive

(MD059, descriptive-link-text)